src/lib/snarky_js_bindings/lib/snarky_js_bindings_lib.ml

```diff
@@ -2455,7 +2455,7 @@ module Ledger = struct
   (* TODO hash two parties together in the correct way *)
 
   let hash_party (p : party) =
-    p |> party |> Party.Preconditioned.digest |> Field.constant |> to_js_field
+    p |> party |> Party.digest |> Field.constant |> to_js_field
```

Comment:
There's a lot of code in here that still needs fixing... like the `body` and `party` functions above (checked + unchecked versions).